docs/ReduceBinarySize.md

```diff
@@ -0,0 +1,37 @@
+# Reduce binary size
+To reduce compiled binary size, two arguments are available:
+
+- --include_ops_by_model=<path to directory of models\>
+- --include_ops_by_file=<path to a json file\>
+
+The arguments empower building script of commenting out operators listed in execution provider(s), thereby downsizing build output.
+
+## include_ops_by_model
+The argument enables the compile binary of including only operators consumed by models in the specified directory.
+
+## include_ops_by_file
+The argument enables the compiled binary of including only operators referred. Json file has format like:
+```
+{
+    "kOnnxDomain":
+    {
```

Comment:
nit: probably shouldn't use internal enum value here. the string for the domain would be clearer for the target audience of external users.

May also be best to avoid listing internal ops like the memcpy ones here (which really shouldn't be in the onnx domain but that's a different matter).

Reply:
Sure, let's switch to non-internal domain strings.

Reply:
Done.